openlibrary/catalog/add_book/__init__.py

```diff
@@ -214,11 +215,11 @@ def add_cover(cover_url, ekey, account=None):
     :rtype: int or None
     :return: Cover id, or None if upload did not succeed
     """
-    olid = ekey.split("/")[-1]
+    olid = ekey.split('/')[-1]
     coverstore_url = config.get('coverstore_url').rstrip('/')
     upload_url = coverstore_url + '/b/upload2'
     if upload_url.startswith("//"):
-        upload_url = "{0}:{1}".format(web.ctx.get("protocol", "http"), upload_url)
+        upload_url = "{0}:{1}".format(web.ctx.get('protocol', 'http'), upload_url)
```

Comment:
Why not change the first four double quotes to be like the last four single quotes?

Reply:
Thanks @cclauss , fixed.